streams/mx.m3u

```diff
@@ -118,6 +120,10 @@ https://stream.oursnetworktv.com/latin/cnrtelevision/playlist.m3u8
 https://stream.oursnetworktv.com/latin/encoder29/playlist.m3u8
 #EXTINF:-1 tvg-id="CreaLaTV.mx",CreaLaTV (720p) [Not 24/7]
 https://ythls.armelin.one/channel/UCL1iKKBxY_iPu7iH-1wNdwQ.m3u8
+#EXTINF:-1 tvg-id="DePeliculaLatinAmerica.mx",De Película Latin America (480p)
+http://kstv.us:8080//live/tH300gvHff/8606073792/46576.m3u8
+#EXTINF:-1 tvg-id="DePeliculaPlus.mx",De Película Plus (720p)
+http://kstv.us:8080//live/tH300gvHff/8606073792/46577.m3u8
```

Comment:
```suggestion
```

Reply:
oh they're xtream? i never noticed
that's sad